package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "autolispext",
 	"displayName": "AutoCAD AutoLISP Extension",
 	"description": "This is a vscode extension for AutoCAD AutoLISP",
-	"version": "1.4.0",
+	"version": "1.3.0",
```

Comment:
1.3.0 is the last official release. `npm version` increments the patch version and appends the preid we specify (commit sha in our case). This results in version like 1.3.1-<sha> which is exactly what we want.